benchmarks/views/user.py

```diff
@@ -226,87 +224,117 @@ def is_submission_original(file, submitter: User) -> Tuple[bool, Union[None, Lis
             identifiers = plugin_has_instances(namelist, plugin)[1]
             db_table = plugin_db_mapping[plugin]
 
-            # Determine the field name based on the plugin type
+            # Determine the lookup field name based on the plugin type
             field_name = 'name' if plugin == "models" else 'identifier'
 
             for identifier in identifiers:
                 query_filter = {field_name: identifier}
 
                 # Check if an entry with the given identifier exists
-                if db_table.objects.filter(**query_filter).exists():
+                if db_table.objects.filter(**query_filter).exists() or "resnet50_tutorial" in identifier:
                     return False, [plugin, identifier]
 
     return True, None  # Passes all checks, then the submission is original -> good to go
 
 
-def validate_zip(file):
+def validate_zip(file: str) -> Tuple[bool, str]:
+    """
+    Validates the structure of a zip file. Checks for the existence of required plugins
+    and their instances, and verifies that each instance contains specific files.
+
+    :param file: Path to the zip file.
+    :return: Tuple containing a boolean for success and an error message if any.
+    """
     with zipfile.ZipFile(file, mode="r") as archive:
         namelist = archive.infolist()
         root = namelist[0]
         has_plugin, submitted_plugins = plugins_exist(namelist)
-        if not has_plugin:
-            return False, f"\nPlease make sure your {root.filename} folder contains at least one of the " \
-                          f"following folders:" \
-                          "[metrics, data, benchmarks, models]"
-        instances = []
-        files = []
-        for plugin in submitted_plugins:
-            has_instance, submitted_instances = plugin_has_instances(namelist, plugin)
-            instances.append(submitted_instances)
-        plugin_instance_dict = dict(zip(submitted_plugins, instances))
-
-        # make sure there is at least one plugin that is not empty:
-        if all(x == [] for x in plugin_instance_dict.values()):
-
-            if len(list(plugin_instance_dict.keys())) == 1:
-                return False, f"\nYour {list(plugin_instance_dict.keys())} folder is empty."
+        if not has_plugin:  # checks for at least one plugin
+            return False, (f"\nPlease make sure your {root.filename} folder contains at least one of the "
+                           "following valid plugin folders: [metrics, data, benchmarks, models].")
```

Comment:
```suggestion
                           "following valid plugin folders: [data, metrics, benchmarks, models].")
```
keep ordering consistent with how we mention it elsewhere